content/themis/languages/python/installation.md

```diff
@@ -75,13 +75,146 @@ Once PyThemis is installed, you can [try out code examples](../examples/).
 If the stable package version does not suit your needs,
 you can manually build and install the latest version of Themis from source code.
 
+### Python wheel
```

Comment:
last comment. let's mention in the paragraph above that there are 2 ways how to install from the sources, building and installing python package from wheel file and building os package in installing system-wide. 

Or just rephrase to `... you can manually build python wheel or .deb/.rpm package and install the latest version of Themis from source code.`

Reply:
Done